packages/manager/apps/container/src/container/common/urls-constants.ts

```diff
@@ -43,12 +45,14 @@ const urls: URLLinks = {
       WS: `${helpRoot}/es${homeIndex}`,
     },
     status: 'https://www.status-ovhcloud.com/',
+    marketplace: 'https://marketplace.ovhcloud.com/',
   },
   US: {
     help: {
       US: 'https://us.ovhcloud.com/support',
     },
     status: 'https://status.us.ovhcloud.com/',
+    marketplace: 'https://marketplace.ovhcloud.com/',
```

Comment:
Currently there is no `marketplace` displayed in CA and US. Are we sure to add it here?